service/src/test/java/bio/terra/externalcreds/services/PassportServiceTest.java

```diff
@@ -202,7 +220,8 @@ class ValidPassportTestParams {
       String visaPhsId = "phs000123";
       String criterionPhsId = visaPhsId;
       boolean persistLinkedAccount = true;
-      Map<String, String> customJwtClaims = Collections.emptyMap();
+      boolean noLinkedAccount = false;
+      Map<String, Object> customJwtClaims = Collections.emptyMap();
```

Comment:
I haven't wrapped my head around this completely but I'm not going to have time the rest of today to continue reviewing this, so this is a bit of an incomplete thought. It feels a bit like we're testing out a test class. Should we be using an instance of whatever this class (`class ValidPassportTestParams`) is mocking out?